src/Nethermind/Nethermind.Evm.Test/Eip6780Tests.cs

```diff
@@ -142,6 +133,56 @@ public void self_destruct_in_same_transaction(long blockNumber, ulong timestamp)
 
             _processor.Execute(createTx, block.Header, NullTxTracer.Instance);
 
+            AssertCodeHash(contractAddress, Keccak.OfAnEmptyString);
+            AssertStorage(new StorageCell(contractAddress, 1), 0);
+
+            TestState.GetBalance(contractAddress).Should().Be(0);
+            TestState.GetBalance(TestItem.PrivateKeyB.Address).Should().Be(99.Ether());
+        }
+
+        [TestCase(MainnetSpecProvider.GrayGlacierBlockNumber, MainnetSpecProvider.CancunBlockTimestamp)]
+        public void self_destruct_in_initcode(long blockNumber, ulong timestamp)
```

Comment:
All tests in this class are copy pasted. Please, reuse the code for the tests a bit ;)

Reply:
Done